Makefile

```diff
@@ -28,7 +28,7 @@ venv:
 deps:
 	@( \
 		source ./$(VIRTUAL_ENV_DIR)/bin/activate; \
-		pip3 install -r requirements.txt; \
+		pip3 install -r requirements.txt --use-feature=2020-resolver; \
```

Comment:
This doesn't work:
```
[sila-postman-signer] (ENGRG-737_python_version_upgrade_to_3.8.6)[...]$ make deps

Usage:
  pip3 install [options] <requirement specifier> [package-index-options] ...
  pip3 install [options] -r <requirements file> [package-index-options] ...
  pip3 install [options] [-e] <vcs project url> ...
  pip3 install [options] [-e] <local project path> ...
  pip3 install [options] <archive url/path> ...

no such option: --use-feature
make: *** [deps] Error 2
```

Reply:
which pip version did you end up using (might be using an older version of `pip`?  Did you run `make venvclean && make venv`?

I've found when upgrading python versions, `python -m venv` won't upgrade an existing virtual_env created with an older version of Python.

Reply:
@estara @johndburk 
checked that --use-feature=2020-resolver no longer has any effect, since it is now the default dependency resolver in pip. I used it with python version 3.8, but there is no need for it with 3.9
Other than this I have rechecked after removing the previous virtual env and everything seemed to work fine

Reply:
@johndburk nope, I did not. Lesson learned. @Arcgate-Pulkit it works great now! Nice job.